features/minion/core/health-check/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,51 @@
+<blueprint
+    xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+        http://www.osgi.org/xmlns/blueprint/v1.0.0 
+        https://osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+">
+
+    <bean id="passiveEchoRpcHealthCheck" class="org.opennms.core.health.api.DefaultPassiveHealthCheck">
```

Comment:
Although you have included `passive` for Echo RPC as well, that doesn't seem to reflect in Rest Client response. It only gives out OpenNMS Rest Response(Passive)

Can you check this ?
![Screen Shot 2021-09-26 at 10 36 04 PM](https://user-images.githubusercontent.com/17052395/134837581-1e8beba5-5e15-45f5-adf8-bb491a7a82db.png)
?

Reply:
Fixed.
`HealthCheckRestServiceImpl` did assemble health check descriptions and health check results in a somewhat complicated and brittle way. Now, health check instances and their execution results a kept toghether.